lib/browser/api/web-contents.js

```diff
@@ -193,7 +194,7 @@ const asyncWebFrameMethods = function (requestId, method, callback, ...args) {
 
 for (const method of webFrameMethods) {
   WebContents.prototype[method] = function (...args) {
-    this._sendInternal('ELECTRON_INTERNAL_RENDERER_WEB_FRAME_METHOD', method, args)
+    ipcMainUtils.invoke(this, 'ELECTRON_INTERNAL_RENDERER_WEB_FRAME_METHOD', method, ...args)
```

Comment:
To avoid confusion can we call this `invokeInRenderer` or `invokeInWebContents` or something to separate it from `ipcRendererUtils`?

Reply:
renamed to `invokeInWebContents`